installer/charts/rhtap-subscriptions/templates/delete_tekton_config.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: delete-tekton-config-if-exists
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  backoffLimit: 3
+  template:
+    spec:
+      serviceAccountName: delete-tekton-config-sa
+      restartPolicy: OnFailure
+      containers:
+        - name: delete-tekton-config
+          image: quay.io/openshift/origin-cli:latest
+          command:
+            - /bin/bash
+            - -c
+            - |
+              echo "Deleting TektonConfig config resource if it exist to allow helm to own the resource"
+              oc delete tektonconfig config --ignore-not-found
```

Comment:
If feel that the TektonConfig should be deleted only if the CLI is managing the subscription and the resource is not annotated as belonging to a helm deployment.

Reply:
Agree with checking if we just installed the subscription. But it won't be a helm deployment cause will only run on install not upgrades. Once install is done, helm should own the CR